### PR TITLE
Fix OSGI version ranges

### PR DESCRIPTION
**Pull request by mbechler**
Use correct format for open ended version ranges.
Also adjust the javax.servet.jsp version (which was falsely set to the javax.servet version)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -82,9 +82,9 @@
         <arquillian.version>1.1.7.Final</arquillian.version>
         <cdi.tck.version>1.2.8.Final</cdi.tck.version>
         <httpclient.version>4.5.2</httpclient.version>
-        <osgi.servlet.range>[2.4,)</osgi.servlet.range>
-        <osgi.el.range>[2.2,)</osgi.el.range>
-        <osgi.faces.range>[2.0,)</osgi.faces.range>
+        <osgi.servlet.range>2.4</osgi.servlet.range>
```
Comment (rmannibucau) [inline pom.xml]: we run with servlet 2.5, 3.0, 3.1 and 4.x so we need a range there I fear, why removing ranges instead of making them "valid"?

**Comment by mbechler** [comment]
These are actually open ended ranges with the respective lower bound, so this is exactly the range intended by the other (invalid) format.

Per spec:
If a version range is specified as a single version, it must be interpreted as the range [version,∞).

**Comment by rmannibucau** [comment]
right, forgot that detail ;)

works for me

**Comment by tandraschko** [comment]
@mbechler the changes seems to be applied. Could you close the PR plz? thanks

**Pull request closed without merging**